### DISCUSSION TITLE
Change the background color of a date.

### DISCUSSION THREAD

**Discussion by mersimoski** [web-ridge/react-native-paper-dates#248] [Q&A] [answered]
Is there a way to change the default color of the date background that is pink-ish ? By default its purple on my application.

<img width="309" alt="image" src="https://user-images.githubusercontent.com/62712311/213899092-b2c2118b-6c9a-4ec3-87d8-514d1c2e3f33.png">

**Comment by iM-GeeKy** [comment, marked as answer]
Hi @mersimoski, all of the colors are derived from your paper [theme](https://callstack.github.io/react-native-paper/theming.html) and if the specific colors are taken from the [material 3 date picker spec](https://m3.material.io/components/date-pickers/specs).

<img width="980" alt="Screenshot 2023-01-22 at 7 15 37 AM" src="https://user-images.githubusercontent.com/7604441/213915277-44e2b1e0-d990-4ade-a528-7fbc2259b2c5.png">

The `primary-container` attribute in your theme is what drives the color of the pink-ish background you are describing.

**Reply by wundertax-usr** [reply]
How should you provide theme color values? I am applying theme via theme prop, but they are not applied to date picker dialog.

```
    <DatePickerInput
                theme={{
                  colors: {
                    primaryContainer: '#00CB9D',
                    primary: '#001233',
                    onPrimary: '#00CB9D', 
                    ...
```
